store/src/java/com/zimbra/cs/service/mail/SendShareNotification.java

```diff
@@ -80,6 +91,12 @@ public class SendShareNotification extends MailDocumentHandler {
     }
 
     private static final String REVOKE = "revoke";
+    private static final String EDIT = "edit";
+    private static final String CREATE = "edit";
+
+    public static final String ZIMBRA_SOAP_USER_AGENT = "ZimbraSync";
```

Comment:
I think this could be `ZimbraWebClient` since the request being triggered from web client for sharing Or better capture the UA from current request and forward it in the further SOAP call.

Reply:
since the method which was using this is  removed, remove these constants